components/channel_members_dropdown/index.ts

```diff
@@ -46,4 +50,4 @@ function mapDispatchToProps(dispatch) {
     };
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(ChannelMembersDropdown);
+export default connect(mapStateToProps, mapDispatchToProps)(ChannelMembersDropdown as React.ComponentClass);
```

Comment:
What's the error you get if you remove the cast for `React.ComponentClass`?

Reply:
By removing the cast for React.ComponentClass the error is - 

Argument of type 'typeof ChannelMembersDropdown' is not assignable to parameter of type 'ComponentType<Matching<{ currentUserId: string; isLicensed: boolean; canChangeMemberRoles: boolean; canRemoveMember: boolean; } & { actions: { getChannelMember: (channelId: string, userId: string) => ActionFunc; getChannelStats: (channelId: string) => ActionFunc; updateChannelMemberSchemeRoles: (channelId: string, u...'.
  Type 'typeof ChannelMembersDropdown' is not assignable to type 'ComponentClass<Matching<{ currentUserId: string; isLicensed: boolean; canChangeMemberRoles: boolean; canRemoveMember: boolean; } & { actions: { getChannelMember: (channelId: string, userId: string) => ActionFunc; getChannelStats: (channelId: string) => ActionFunc; updateChannelMemberSchemeRoles: (channelId: string, ...'.
    Types of parameters 'props' and 'props' are incompatible.
      Type 'Matching<{ currentUserId: string; isLicensed: boolean; canChangeMemberRoles: boolean; canRemoveMember: boolean; } & { actions: { getChannelMember: (channelId: string, userId: string) => ActionFunc; getChannelStats: (channelId: string) => ActionFunc; updateChannelMemberSchemeRoles: (channelId: string, userId: string,...' is not assignable to type 'Props'.
        The types returned by 'actions.removeChannelMember(...)' are incompatible between these types.
          Type 'ActionFunc' is not assignable to type 'string'.ts(2345)